website/src/pages/news.tsx

```diff
@@ -0,0 +1,103 @@
+                                        <li><strong>Security Boulevard</strong> November 28, 2019<br />
+                                        <a href="https://securityboulevard.com/2019/11/5-free-developer-tools-were-grateful-for/" rel="nofollow">5 Free Developer Tools We’re Grateful For</a></li>
+                                        <li><strong>DevClass</strong> November 25, 2019<br />
+                                        <a href="https://devclass.com/2019/11/25/gitlab-12-5-gets-multi-cloud-rolling/" rel="nofollow">GitLab 12.5 gets multi-cloud rolling</a></li>
+                                        <li><strong>App Developer Magazine </strong> November 28, 2019<br />
+                                        <a href="https://appdevelopermagazine.com/universal-source-code-search-engine-emerges/" rel="nofollow">Universal source code search engine emerges</a></li>
+                                        <li><strong>Dataversity</strong> November 18, 2019<br />
+                                        <a href="https://www.dataversity.net/sourcegraph-announces-new-gitlab-native-integration-universal-code-search-engine/" rel="nofollow">Sourcegraph Announces New GitLab Native Integration, Universal Code Search Engine</a></li>
+                                        <li><strong>The New Stack</strong> November 15, 2019<br />
+                                        <a href="https://thenewstack.io/this-week-in-programming-dear-tech-conferences-do-better/" rel="nofollow">This Week in Programming: Dear Tech Conferences, Do Better</a></li>
+                                        <li><strong>DevClass</strong> November 13, 2019<br />
+                                        <a href="https://devclass.com/2019/11/13/gitlab-gets-native-with-sourcegraph-code-review-in-upcoming-release/" rel="nofollow">GitLab gets native with Sourcegraph code review in upcoming release</a></li>
+                                        <li><strong>SD Times </strong> November 12, 2019<br />
+                                        <a href="https://sdtimes.com/softwaredev/sd-times-news-digest-codefreshs-ci-cd-live-debugger-azure-functions-premium-and-smartbear-announces-readyapi-3-0/" rel="nofollow">SD Times news digest: Codefresh’s CI/CD live debugger, Azure Functions Premium, and SmartBear announces ReadyAPI 3.0</a></li>
+                                    </ul> 
+                                </div>
+                            </div>
+                        </div>  
+
+                        <div class="container">
+                            <div class="row justify-content-md-center">
+                                <div class="col mt-5">                                           
+                                    <h3>Media Contact</h3>
```

Comment:
```suggestion
                                    <h3>Media contact</h3>
```

per our style guide (https://about.sourcegraph.com/handbook/communication/style_guide#general): "Use 'Sentence case'"